### PR TITLE
Restore mistakenly changed instances of 'bitcoin'

### PR DESCRIPTION
**Pull request by Scotty0448**
This restores instances of the word 'bitcoin' that were mistakenly changed to 'RavenProject', 'Ravencoin' and 'raven' when initially forking Bitcoin's codebase.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and performance improvements, as well as updated translations.
 
 Please report bugs using the issue tracker at GitHub:
 
-  <https://github.com/RavenProject/Ravencoin/issues>
```
Comment (TronBlack) [inline doc/release-notes.md:13]: This seems like it should still be RavenProject/Ravencoin

**Review by TronBlack — CHANGES_REQUESTED** [review]
One change for where to submit issues.

Thanks for your work on this.

File: doc/release-notes.md (hunk 1 above)
Comment (Scotty0448) [reply doc/release-notes.md]: Yep, my mistake while changing the others which referenced specific past issues.

**Review by fdoving — APPROVED** [review]
ACK 558de8177a00f7e8c91c8b7c6239096140080fbf

**Review by HyperPeek — APPROVED** [review]

**Pull request closed without merging**